wallet/src/repository/wallet/mod.rs

```diff
@@ -328,16 +329,7 @@ where
 
         // Total amount of state and db transactions
         let total = state.transaction_next_id + u32::try_from(state.local_movements.len()).unwrap();
```

Comment:
What's the difference between `total` and `db_total`? `db_total` is obtained from the database, but shouldn't that be equal to `state.transaction_next_id`?

Reply:
exactly! :)